README.md

```diff
@@ -20,6 +20,36 @@ var Writable = zstreams.Writable,
 The base zstreams function like, and inherit from, the native Node.JS streams.  They mostly behave like their
 parents, with
 
+### Simplified Constructors
+
+Zstreams supports iojs simplified stream constructors as seen [here](https://iojs.org/api/stream.html#stream_simplified_constructor_api).
+
+Additionally the `_flush` method has been added to writable and duplex streams.
```

Comment:
This line doesn't belong in this section of the readme, and should probably be moved.